src/main/java/Parser.java

```diff
@@ -0,0 +1,170 @@
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.Todo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+
+public class Parser {
+
+    /**
+     * Returns the task number (as indexed in the task list)
+     * of the task to be deleted or marked as done.
+     *
+     * @param keyword indicates if the task is to be deleted or marked as done.
+     * @param task task to be deleted or marked as done (required).
+     * @return index of the required task in the task list.
+     */
+    private static int getTaskNumber(String keyword, String task) {
```

Comment:
JavaDoc header comemnts are actually not needed for private methods, just thought you might want to know